path_rotate.go

```diff
@@ -127,7 +127,7 @@ func (b *backend) pathRotateRoleCredentialsUpdate(ctx context.Context, req *logi
 	// In create/update of static accounts, we only care if the operation
 	// err'd , and this call does not return credentials
 	item, err := b.popFromRotationQueueByKey(name)
-	if err != nil {
+	if item == nil || err != nil {
```

Comment:
How are we hitting a case when `item == nil` but `err == nil`? Looking at https://github.com/hashicorp/vault/blob/e1d21dc601ced5bd925510ee2fe8238cee67781e/builtin/logical/database/rotation.go#L536 it seems like if `item` is ever `nil` we'll definitely have an error. i.e. the code path would never be different than what it is today?

Reply:
Good catch! It was actually due to `resp` being nil and `item` was trying to use a value from it.  Fixed in https://github.com/hashicorp/vault-plugin-secrets-openldap/pull/14/commits/e782ca24a52aa477aad736d5ce30a206b8c55fab